src/formatting/lists.rs

```diff
@@ -482,19 +481,88 @@ where
                 let width = formatting.shape.width.checked_sub(overhead).unwrap_or(1);
                 let offset = formatting.shape.indent + overhead;
 
-                let comment_shape = Shape::legacy(width, offset);
```

Comment:
@calebcartwright I rebased with the master branch at the earlier commit. Accidentally I worked in the same commit that resolving the conflicts. L485-L565 lines from the last conflicted commit.

Reply:
Cheers. That's not too surprising given the age, though I'll need to squash anyway give the merge commit.

I _should_ finally have a bit of bandwidth in the next day or two to get back to the PR queue so will hopefully be able to review soon.

For future reference/PRs though it'd be helpful if to just do a clean rebase of your topic branch on the upstream branch from this repo and avoid merge commits :+1: Thanks again for all your work!